WordPress/Sniffs/Security/NonceVerificationSniff.php

```diff
@@ -413,6 +413,8 @@ private function set_cache( $filename, $start, $end, $nonce ) {
 	 */
 	protected function mergeFunctionLists() {
 		if ( $this->customNonceVerificationFunctions !== $this->addedCustomNonceFunctions ) {
+			$this->customNonceVerificationFunctions = array_map( 'strtolower', $this->customNonceVerificationFunctions );
+
```

Comment:
Why are you changing the user-provided value ? instead of changing the merge result `$this->nonceVerificationFunctions` ?

Reply:
I had in my mind that the user-provided value is the only one that required change, and that is why I opted to change it. After seeing your question, I imagine you are suggesting changing `$this->nonceVerificationFunctions` to ensure the sniff behaves correctly if, in the future, a new function is added to this property without all lowercase letters, and also to ensure it behaves correctly for sniffs extending this one. Is that the case?

I went ahead and pushed a new commit changing `$this->nonceVerificationFunctions` instead of `$this->customNonceVerificationFunctions`.